src/element.toit

```diff
@@ -34,6 +35,22 @@ abstract class Element implements Window:
   x -> int?: return x_
   y -> int?: return y_
 
+  /**
+  Constructs an Element.
+  The x and y coordinates are relative to the parent element.
+  The style can be used to apply a custom $Style object to this element
+    alone.  Normally, you would apply a style to the whole tree of elements
+    using $PixelDisplay.set_styles method.
+  The $element_class is a string that can be used to identify the element
```

Comment:
Do we even want to have the `element_class`?
It's longer than `classes` (even with the additional `[]`) and doesn't seem to have any advantage.